Libraries/IntegrationServiceLibrary.psm1

```diff
@@ -298,12 +298,12 @@ function Wait-ForVMToStartKVP {
 	param (
 		[String] $VmName,
 		[String] $HvServer,
-		[int] $StepTimeout
+		[int] $StepTimeout = 10
 	)
 	$ipv4 = $null
 	$retVal = $False
 
-	$waitTimeOut = $StepTimeout
+	$waitTimeOut = ($StepTimeout,10 | Measure -Min).Minimum
```

Comment:
So, following command -
```Get-IPv4AndWaitForSSHStart -VmName $VMName -HvServer $HvServer -Vmport $VMPort `
        -Password $VMPassword -User $VMUserName -StepTimeout 1000```
will result in timeout of 10 minutes?

It looks like, we are not respecting the parameters passed to this function. 
While I agree that, 1000 minutes timeout is insane, we should fix the timeout in https://github.com/LIS/LISAv2/blob/ab40635dde97f9790b72a8214b5abe1cf744ad40/Testscripts/Windows/STRESS-MTU-Netvsc-Reload.ps1#L70 

... rather than changing the function.

So I suggest to, fix the timeout values passed to this function

Reply:
@iamshital Okay, we will fix the -StepTimeout in passed places, we missed this stress case.